conda-recipe/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set package_name = "pmps-ui" %}
+{% set import_name = "pmpsui" %}
+{% set version = load_file_regex(load_file=os.path.join(import_name, "_version.py"), regex_pattern=".*version = '(\S+)'").group(1) %}
+
+package:
+  name: {{ package_name }}
+  version: {{ version }}
+
+source:
+  path: ..
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+
+
+requirements:
+  build:
+  - python >=3.7
+  - pip
+  - setuptools_scm
+  run:
+  - python >=3.7
+  - pcdsutils
+  - prettytable
+  - pydm
+  - pyqt
```

Comment:
I'm not sure if this is only a conda thing or a general thing, but we need `pyqtwebengine` installed too. Starting at a certain version of pyqt, the web engine stuff isn't bundled with the main pyqt package...
```
ImportError: cannot import name 'QtWebEngineWidgets' from 'PyQt5' (/cds/home/z/zlentz/miniconda3/envs/pmps-ui-test/lib/python3.9/site-packages/PyQt5/__init__.py)
```

Reply:
I kinda just assumed this was in PyDM.  Whoops